cidrset/cidr_set.go

```diff
@@ -61,6 +65,10 @@ var (
 	// big compared to the CIDR mask size.
 	ErrCIDRSetSubNetTooBig = errors.New(
 		"New CIDR set failed; the node CIDR size is too big")
+	// ErrSubNetMaskSizeInvalid occurs when the subnet mask size  is invalid
+	// big than 32 for ipv4 and big than 128 for ipv6
+	ErrSubNetMaskSizeInvalid = errors.New(
+		"SubNetMask is invalid, should be less than 32 for ipv4 and less than 128 for ipv6")
```

Comment:
Better reuse the constants we've set above:
```suggestion
	// ErrSubNetMaskSizeInvalid occurs when the subnet mask size is invalid:
	// bigger than 32 for IPv4 or bigger than 128 for IPv6
	ErrSubNetMaskSizeInvalid = fmt.Errorf(
		"SubNetMask is invalid, should be lower or equal to %d for IPv4 and to %d for IPv6",
		maxSubNetMaskSizeIPv4, maxSubNetMaskSizeIPv6)
```

Reply:
done